### ISSUE TITLE
RealEstatePro: Call to undefined function get_field() in header.php

### ISSUE THREAD

**Issue by livingwandsworth** [Markup360/realestatepro#14]
-I used eHost to register my domain. I have already installed Wordpress. I think I have Wordpress version 4.7.2.
-I logged into my Wordpress Admin account and uploaded RealEstatePro.zip, [after downloading it from your site] from the Appearance > Themes > Add New > Upload Theme.
-Now I was able to see RealEstatePro Theme, under Appearance > Themes.
-I clicked on 'Live Preview' button to test.
-I get a Fatal error message, 'Fatal error: Call to undefined function get_field() in /home2/dreamsco/public_html/wp-content/themes/realestatepro/header.php on line 5'
-I have not installed MySQL Database, nor did I copy the files, as you show in your Youtube video 'https://www.youtube.com/watch?v=gwiFhx5p5nQ'

I like your Theme, your video and find that you also have a Forum to post questions. Please assist in helping resolve this Fatal Error.

thanks in advance,
HomBuildr 

-Originally asked by Hom177 on 02-03-2017, 04:13 AM, migrated from site forum

**Comment by livingwandsworth** [comment]
-Looks like I have installed most of the Plugins, except 'Pirate Forms'. I am thinking it's not for RealEstatePro or Graviton. Do you want me to install it?

thanks,
HomBuildr.

**Comment by livingwandsworth** [comment]
-Looks like I have installed most of the Plugins, except 'Pirate Forms'. I am thinking it's not for RealEstatePro or Graviton. Do you want me to install it?

-I get similar error, when installing m0-graviton.zip as well.  The Fatal Error is 'Fatal error: Call to undefined function get_field() in /home2/dreamsco/public_html/wp-content/themes/graviton/header.php on line 5'

-Please help.

thanks,
HomBuildr.

**Comment by Markup360** [comment]
After you installed the theme you would have received a message at the top to install the plugins that come with RealEstatePro, have you installed them?

**Comment by livingwandsworth** [comment]
-Looks like I have installed most of the Plugins, except 'Pirate Forms'. I am thinking it's not for RealEstatePro or Graviton. Do you want me to install it?

thanks,
HomBuildr.

**Comment by livingwandsworth** [comment]
-Looks like I have installed most of the Plugins, except 'Pirate Forms'. I am thinking it's not for RealEstatePro or Graviton. Do you want me to install it?

-I get similar error, when installing m0-graviton.zip as well.  The Fatal Error is 'Fatal error: Call to undefined function get_field() in /home2/dreamsco/public_html/wp-content/themes/graviton/header.php on line 5'

-Please help.

thanks,
HomBuildr.

**Comment by Markup360** [comment]
No you don't need to install that, is the Advanced Custom Fields plugin installed?

**Comment by livingwandsworth** [comment]
Naz,
I don't see any other plugin other than 'Pirate Forms', to be installed. All Plugins have been installed. Are you referring to 'Contact Form 7' or 'Page Builder by SiteOrigin'. Both plugins have been installed and I was still getting the error.

thanks,
HomBuildr

**Comment by Markup360** [comment]
No when you install RealEstatePro you should get a message to install 7 plugins, see this part of the RealEstatePro video, look at the top to see the notice.

https://youtu.be/vvx27YCyUZo?t=7m51s

**Comment by livingwandsworth** [comment]
Naz,
I got the error mentioned, when I did 'Live Preview'. If you do not Activate the Theme, you do not see the 7 Plugins. I went ahead and Activated the Theme, then I was able to see the request for the 7 Plugins. I installed the 7 Plugins and the Fatal error has been resolved. I do not get the error now.
-Is RealEstatePro, already using Graviton? Will I be able to watch your videos for both RealEstatePro and Graviton and build the pages as shown in those videos?

thank you,
HomBuildr.
